pythonforandroid/build.py

```diff
@@ -549,7 +552,10 @@ def build_recipes(build_order, python_modules, ctx, project_dir,
             ('The requirements ({}) were not found as recipes, they will be '
              'installed with pip.').format(', '.join(python_modules)))
 
-    recipes = [Recipe.get_recipe(name, ctx) for name in build_order]
+    recipes = [
+        Recipe.get_recipe(name, ctx)
+        for name in build_order
+    ]
```

Comment:
Nitpick: At some point we may introduce black and this will move again which introduces noise.
Basically since that line wasn't already 79 chars, black would just make it one line again I suppose.

Reply:
good point, removed it!